spec/support/bullet.rb

```diff
@@ -3,6 +3,7 @@
     config.before(:each) do
       Bullet.add_whitelist type: :unused_eager_loading, class_name: "User", association: :organisation
       Bullet.add_whitelist type: :unused_eager_loading, class_name: "Activity", association: :organisation
+      Bullet.add_whitelist type: :unused_eager_loading, class_name: "Activity", association: :parent
```

Comment:
This makes me a bit nervous - mostly because I don't fully understand bullet!

Can we disable bullet around the tests we know it is a false positive rather than for the whole `Activity` class?

Reply:
It makes me nervous too because it might hide alerts about other places we are eager loading but not using it, which may cause things to slow down unnecessarily. 

I'll take another look and see what an alternative might look like and report back here 👍

Reply:
I `think` we could diasble bullet around the specs: https://github.com/flyerhzm/bullet#run-in-tests